### PR TITLE
chore: start new welcome flow

### LINKED ISSUE #1504: Onboarding flow improvements
### Is your feature request related to a problem? Please describe

The onboarding flow has some flaws that we want to address with this issue.

#### Remove benefits page

 - Users might not see the "Get started" button at the very top
 - Users do not read this page

image

#### Remove progress bar
image

#### Remove the page where we ask people to send us SATS.
 - Most of our users don't have any sats in their wallet
 - Don't ask them to spend (will add a "fund you wallet" guide later)

image

#### Questions
 - Where should we send users after they have setup their account? Should we maybe open the extension startpage? 
image

### PR DESCRIPTION
**Pull request by bumi**
Fixes #1504

https://user-images.githubusercontent.com/64399555/194258432-ceba0956-df51-4950-82f4-23e5c0756e1b.mp4

### CODE REVIEW

**Comment by escapedcat** [comment]
image
Do we still want the success-screen when adding an account even after the onboarding? No, right? @dvoroneca @reneaaron

**Comment by reneaaron** [comment]
Considering the feedback that no-one really used the tutorial page I think this is an improvement. I'd still vote for removing the "your websites" section if there are none within this PR. 

image

This page needs some improvement though: @dvoroneca 
image
 - The question is wrong. What if I don't have a Lightning wallet already? 
 - Emphasize Alby Accounts for first-time users 

If you agree, can you please make sure we consider those points in the follow-up-issue?

File: src/app/screens/Onboard/TestConnection/index.tsx [hunk 1]
```diff
@@ -33,14 +26,8 @@ export default function TestConnection() {
...
-      const response = await api.getAccountInfo();
-      const name = response.name;
-      const { alias } = response.info;
-      const { balance: resBalance } = response.balance;
-      const balance =
-        typeof resBalance === "number" ? resBalance : parseInt(resBalance);
-
-      setAccountInfo({ alias, balance, name });
+      await api.getAccountInfo();
+      utils.redirectPage("options.html");
```
Comment (bumi) [inline src/app/screens/Onboard/TestConnection/index.tsx:30]: can we explicitly check that we got valid data from the `getAccountInfo` call and only then redirect? 

this page is about testing the connection so lets also explicitly check this in the code.

**Comment by bumi** [comment]
I think it would be better to use a link that we control, then it is easier for us to change it. 
e.g. getalby.com/topup 

If we don't have content there we can redirect from there to the guide.

**Comment by reneaaron** [comment]
I think we can improve the text here: 

image

 - "Your bitcoin lightning payments on the web" seems a bit lost in there and doesn't help the user. I'd probably just remove this paragraph.
 - "To start using Alby you might want to get some sats in the wallet" (we are speaking to a first time user here, I'd suggest removing sats) ➡️ To start using Alby you first need to top up your account with bitcoin. Learn how

**Comment by escapedcat** [comment]
Updated text & link:  
image

**Comment by escapedcat** [comment]
@bumi 

> I think it would be better to use a link that we control, then it is easier for us to change it. e.g. getalby.com/topup
> 
> If we don't have content there we can redirect from there to the guide.

[Currently we have moonpay under `/topup`](https://github.com/getAlby/getalby.com/blob/master/config/routes.rb#L20-L21). Which is only reachable for Alby accounts.  

Should we redirect this to our gitbook?
```rb
get '/topup', to: redirect('https://guides.getalby.com/overall-guide/alby-lightning-wallet/how-to-top-up-your-wallet-balance')
```

**Comment by MoritzKa** [comment]
We can use this link now https://guides.getalby.com/overall-guide/alby-lightning-wallet/how-to-top-up-your-wallet-balance, yes.

**Comment by escapedcat** [comment]
> We can use this link now https://guides.getalby.com/overall-guide/alby-lightning-wallet/how-to-top-up-your-wallet-balance, yes.

Created a PR here: https://github.com/getAlby/getalby.com/pull/257

File: src/app/screens/Publishers/index.tsx [hunk 2]
```diff
@@ -90,21 +90,40 @@ function Publishers() {
...
+        <div className="mt-12 bg-white dark:bg-surface-02dp shadow-md px-6 py-8 rounded-lg w-full">
+          <h2 className="mb-2 text-2xl font-bold dark:text-white">
+            {t("unused.title")}
+          </h2>
+
+          <p className="mb-2 dark:text-white">
+            {t("unused.description")}
+
+            <br />
+            <a
```
Comment (escapedcat) [inline src/app/screens/Publishers/index.tsx:117]: Also add: `https://getalby.com/topup`

**Comment by escapedcat** [comment]
Status:
image

**Comment by escapedcat** [comment]
> This ok for a first iteration till we start with https://github.com/getAlby/lightning-browser-extension/issues/1845 ?

#1845 is faster then this. Will wait till it's merged and rework the onboarding-flow after.

**Comment by escapedcat** [comment]
Updated this PR
- merged latest master and adapted the ee2 tests
- removed the info box because this will be tackled by:  
     #1892

**Review by rolznz — APPROVED** [review]

**Review by reneaaron — APPROVED** [review]
tACK, let's get #1892 merged too! 🚀

**Comment by escapedcat** [comment]
- resolved conflicts
- e2e test is not related to this, also fails on master. Will have a look.  
    `successfully connects to Core Lightning` fails

**Pull request merged**